Form/Form.js

```diff
@@ -0,0 +1,11 @@
+document.querySelector('.form-button').addEventListener('click', (e) => {
+    e.preventDefault();
+    const data = {
+        header: document.querySelector('.form-title').value,
+        date: 'January 30th 2018',
```

Comment:
I was wondering why the date was the 30th.

Reply:
I hard coded the date in initially and was supposed to go back in and look for a Date method. IIwill refactor this over the weekend.

Reply:
```suggestion
        date: 'January 30th 2018',
```
`${d.toLocaleDateString('en-GB', { month: 'short' })}
            ${d.getDay()}th,
            ${d.getFullYear()}`